### PR TITLE
Doctrine coding standard 10

### PR DESCRIPTION
**Pull request by gsteel**
|    Q          |   A
|-------------- | ------
| QA            | yes

### Description

Supersedes #206

### CODE REVIEW

File: src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php [hunk 1]
```diff
@@ -13,77 +13,74 @@ final class GetMilestoneFirst100IssuesAndPullRequests implements GetGithubMilest
...
-          author {
-            login,
-            url
-          },
-          labels (first: 100) {
-            nodes {
-              color,
-              name,
-              url
+        query GetStuff($owner: String!, $repositoryName: String!, $milestoneNumber: Int!) {
```
Comment (Ocramius) [inline src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php:16]: Would probably keep this as it was before: I don't trust anything in GraphQL, after some bad experiences with it :D

Comment (Ocramius) [reply src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php:16]: Ah, wait, did the indentation change, or just the nowdoc wrapper?

Comment (gsteel) [reply src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php:16]: Just the indent

Comment (Ocramius) [reply src/Github/Api/GraphQL/Query/GetMilestoneFirst100IssuesAndPullRequests.php:16]: I'll take the risk :D 

Really don't like the new NOWDOC syntax, but :shrug:

**Review by Ocramius — APPROVED** [review]
:ship:

**Pull request merged**